num_masks: 1
mask_1:
  low: [6, 23, 48]
  high: [25, 220, 94]
